public/cli/lib/commands/ffmpeg.integration.ts

```diff
@@ -0,0 +1,36 @@
+import test from 'ava';
+import fs from 'fs';
+import { mergeWavFiles } from './ffmpeg';
+
+let _ffmpegPath = '';
+function getFFMpegPath(): string {
+  if (_ffmpegPath == '') {
```

Comment:
This can be simplified using tenary operator.

Reply:
Yes, however I am expecting there to be multiple lines in the "complicated setup" part, which is why there is an explicit if statement here.  Future work.  If it turns out that this can be a one-liner, I will go for that.  Thanks.